pynest/nest/tests/test_NodeCollection.py

```diff
@@ -715,8 +715,8 @@ def test_empty_nc(self):
         with self.assertRaises(nest.kernel.NESTErrors.BadProperty):
             nest.Connect(empty_nc, empty_nc)
 
-        self.assertTrue(empty_nc.empty())
-        self.assertFalse(nodes.empty())
+        self.assertFalse(bool(empty_nc))
```

Comment:
would ``self.assertFalse(empty_nc)`` work? (it should)
if so please switch all tests

Reply:
`assertFalse()` uses `if expr:`, and `assertTrue()` uses `if not expr:`, so both work.